src/tcgwars/logic/impl/gen3/Deoxys.groovy

```diff
@@ -297,11 +315,7 @@ public enum Deoxys implements LogicCardInfo {
             energyCost R, C, C
             attackRequirement {}
             onAttack {
-              multiSelect(opp.all, 2).each{
-                targeted(it){
-                  damage 30, it
-                }
-              }
+              multiSelect(opp.all, 2).each{ damage 30, it }
```

Comment:
I think this can even be simplified down to this:
```suggestion
              damage 30, multiSelect(opp.all, 2)
```
due to the damage override that takes a list of PCS

Reply:
Do you have any examples of it being used that way already? Looks the same way to me and I'm up for trying (looks cleaner as well), but I'd rather test it somewhere else as a single PR, and then do a mass correction across sets :P

Reply:
Quite a few it seems. https://gist.github.com/flagrama/1bfe737d82f9a1a9deedd9eb06a7c2d4 though none that explicitly use multiSelect.

Reply:
I don't think it's currently being used, no.

Reply:
`damage 30, multiSelect(opp.all, 2)` would not work :)